app/views/languages/not_found.erb

```diff
@@ -11,7 +11,7 @@
 
       <p>
         If you'd like to help implement it, then check out the
-        <a href="https://github.com/exercism/x-common/pull/125">Contributing Guide</a>, which
+        <a href="https://github.com/exercism/problem-specifications/pull/125">Contributing Guide</a>, which
```

Comment:
This Let's link to this instead: https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/README.md